shared_files/baseDstu3/Coverage

```diff
@@ -0,0 +1 @@
f5b-4a613bbae83d","meta":{"lastUpdated":"2018-10-16T15:23:01.648-04:00"},"type":"searchset","total":3,"link":[{"relation":"self","url":"https://fhir.backend.bluebutton.hhsdevcloud.us/baseDstu3/Coverage/?_format=application%2Fjson%2Bfhir&beneficiary=20000000000001"}],"entry":[{"fullUrl":"https://fhir.backend.bluebutton.hhsdevcloud.us/baseDstu3/Coverage/part-a-20000000000001","resource":{"resourceType":"Coverage","id":"part-a-20000000000001","extension":[{"url":"https://bluebutton.cms.gov/resources/variables/ms_cd","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/ms_cd","code":"10","display":"Aged without end-stage renal disease (ESRD)"}},{"url":"https://bluebutton.cms.gov/resources/variables/orec","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/orec","code":"0","display":"Old age and survivor’s insurance (OASI)"}},{"url":"https://bluebutton.cms.gov/resources/variables/crec","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/crec","code":"0","display":"Old age and survivor’s insurance (OASI)"}},{"url":"https://bluebutton.cms.gov/resources/variables/esrd_ind","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/esrd_ind","code":"0","display":"the beneficiary does not have ESRD"}},{"url":"https://bluebutton.cms.gov/resources/variables/a_trm_cd","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/a_trm_cd","code":"0","display":"Not Terminated"}}],"status":"active","type":{"coding":[{"system":"Medicare","code":"Part A"}]},"beneficiary":{"reference":"Patient/20000000000001"},"grouping":{"subGroup":"Medicare","subPlan":"Part A"}}},{"fullUrl":"https://fhir.backend.bluebutton.hhsdevcloud.us/baseDstu3/Coverage/part-b-20000000000001","resource":{"resourceType":"Coverage","id":"part-b-20000000000001","extension":[{"url":"https://bluebutton.cms.gov/resources/variables/ms_cd","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/ms_cd","code":"10","display":"Aged without end-stage renal disease (ESRD)"}},{"url":"https://bluebutton.cms.gov/resources/variables/b_trm_cd","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/b_trm_cd","code":"0","display":"Not Terminated"}}],"status":"active","type":{"coding":[{"system":"Medicare","code":"Part B"}]},"beneficiary":{"reference":"Patient/20000000000001"},"grouping":{"subGroup":"Medicare","subPlan":"Part B"}}},{"fullUrl":"https://fhir.backend.bluebutton.hhsdevcloud.us/baseDstu3/Coverage/part-d-20000000000001","resource":{"resourceType":"Coverage","id":"part-d-20000000000001","extension":[{"url":"https://bluebutton.cms.gov/resources/variables/ms_cd","valueCoding":{"system":"https://bluebutton.cms.gov/resources/variables/ms_cd","code":"10","display":"Aged without end-stage renal disease (ESRD)"}}],"status":"active","type":{"coding":[{"system":"Medicare","code":"Part D"}]},"beneficiary":{"reference":"Patient/20000000000001"},"grouping":{"subGroup":"Medicare","subPlan":"Part D"}}}]}
```

Comment:
All three of these files should be clearly marked as sample data (with no PII/PHI). I'd suggest:

1. Adding a `README.md` to the same directory that explains where they came from (and how to regenerate them, if needed).
2. Renaming the directory "`sample_backend_synthetic_results`" (or some hopefully-less-wordy version of that).

Reply:
The file paths are mirroring the Blue Button url pattern so I can use the path variable to find the files.  I will add a readme.md to this dir explaining what is going on.

Reply:
I added the read me and also made new copies of these for the new BB url schema.